src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -455,11 +452,22 @@ export default {
 		padding: 0;
 		padding-right: $icon-margin;
 		white-space: nowrap;
-		color: var(--color-text-light);
+		color: var(--color-main-text);
 		background-repeat: no-repeat;
 		background-position: $icon-margin center;
 		background-size: $icon-size $icon-size;
 		line-height: $clickable-area;
+		&:hover,
+		&:hover ~ .app-navigation-entry__utils,
+		&:focus,
+		&:focus ~ .app-navigation-entry__utils {
+			background-color: var(--color-background-hover);
```

Comment:
The --color-background-hover needs to be added to css-variables.scss first as well for this to work. :)

Reply:
done :)